app/stylesheets/layout.scss

```diff
@@ -30,7 +30,7 @@ body {
   left: 0;
   position: absolute;
   right: 0;
-  top: $header-height + $filters-closed-height;
+  top: $header-height;
```

Comment:
we still need `+ $filters-close-height` for when the filters are showing. Maybe we need a `.with-filters` class on the waterpoints `.main` div to add it or something